scripts/ImportExportTest.sh

```diff
@@ -0,0 +1,302 @@
+#!/usr/bin/env bash
+
+set -e
```

Comment:
Would this script work with these?
```suggestion
set -euo pipefail
```

I wanted to do that last time I was going through these scripts but I did not have time to test each one individually and make sure it does not break anything. Now that you're rewriting it, it looks like a good opportunity to try this.